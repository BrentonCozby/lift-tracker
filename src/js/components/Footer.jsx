import React from 'react'

const Footer = () => (
    <footer className="Footer">
        <p className="copyright">&copy; 2017. Brenton Cozby. View more projects at <a href="https://brentoncozby.com">brentoncozby.com</a>.</p>
    </footer>
)

export default Footer
