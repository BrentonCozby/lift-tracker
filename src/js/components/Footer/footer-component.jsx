import React from 'react'

export default function Footer() {
    return (
        <footer className="Footer">
            <p className="copyright">&copy; 2017. Brenton Cozby. View more projects at <a href="https://brentoncozby.com">brentoncozby.com</a>.</p>
        </footer>
    )
}
