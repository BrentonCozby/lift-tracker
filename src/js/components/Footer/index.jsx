import React from 'react'

export default () => (
    <footer className="Footer">
        <p className="copyright">&copy; 2017. Brenton Cozby. View more projects at <a href="https://brentoncozby.com">brentoncozby.com</a>.</p>
    </footer>
)
