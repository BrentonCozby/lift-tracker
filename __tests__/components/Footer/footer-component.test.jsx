import React from 'react'
import renderer from 'react-test-renderer'
import Footer from '../../../src/js/components/Footer/footer-component.jsx'

function setup() {
    const props = {
        
    }

    const wrappedApp = renderer.create(<Footer {...props} />)

    return {
        props,
        wrappedApp
    }
}

describe('Footer', () => {
    it('should render self and subcomponents', () => {
        const { wrappedApp } = setup()

        expect(wrappedApp.toJSON()).toMatchSnapshot()
    })
})