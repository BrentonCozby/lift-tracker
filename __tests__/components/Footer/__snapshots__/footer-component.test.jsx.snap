// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render self and subcomponents 1`] = `
<footer
  className="Footer"
>
  <p
    className="copyright"
  >
    Â© 2017. Brenton Cozby. View more projects at 
    <a
      href="https://brentoncozby.com"
    >
      brentoncozby.com
    </a>
    .
  </p>
</footer>
`;
